drivers/hmis/lib/tasks/ce_starter_pack_20250302.rake

```diff
@@ -201,6 +201,7 @@ task ce_starter_pack_20250302: [:environment] do
   start_workflow_event = create_start_event(enrollment_creator_template, 'start referral', 'start_workflow', 'start_referral')
   accept_workflow_event = create_end_event(enrollment_creator_template, 'accept referral', 'end_workflow', 'accept_referral')
 
+  client_acceptance_task = create_task(client_accepts_form_def, enrollment_creator_template,  'Confirm Client Accepts Referral', case_managers)
```

Comment:
These changes to the starter pack update the Enrollment Creator template to have multiple steps in progress at the same time, just to show what that looks like. 
![Screenshot 2025-05-06 at 6 27 35 PM](https://github.com/user-attachments/assets/04f683f1-9df4-49eb-bafc-2f4a8adf3a96)